### PR TITLE
add configmap with iptables

### PR DESCRIPTION
**Pull request by MarinaShustowa**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -32,6 +32,8 @@ import (
 	"syscall"
 	"time"
 
+	"knative.dev/pkg/configmap"
```
Comment (denis-tingaikin) [inline main.go:35]: Why do we need this?

File: main.go [hunk 2]
```diff
@@ -326,20 +329,17 @@ func getNseEndpoint(config *Config, listenOn fmt.Stringer) *registryapi.NetworkS
 	return nse
 }
 
-func getSetIPTablesRulesServerChainElement() networkservice.NetworkServiceServer {
```
Comment (denis-tingaikin) [inline main.go:329]: Should we use thids by default?

File: main.go (hunk 1 above)
Comment (MarinaShustowa) [reply main.go:35]: This allows loading content from configmap in map[string]string form without implementing our own reader.

Comment (denis-tingaikin) [reply main.go:35]: Coulld we simple deserialize rules via yaml parser?

**Pull request closed without merging**